<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsWidget</class>
 <widget class="QWidget" name="StatsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>211</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Item</string>
   </property>
   <widget class="QLabel" name="item3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="item1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="item2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="item4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="item5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="item6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="item7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Enemy</string>
   </property>
   <widget class="QLabel" name="enemy1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemy2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemy3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemy4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemy5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemy10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemy9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemy8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemy7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemy6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>480</y>
     <width>97</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
